src/pattern-library/components/patterns/ContainerPatterns.js

```diff
@@ -1,9 +1,9 @@
 import { useState } from 'preact/hooks';
 
-import Library from '../Library';
-
 import { IconButton, LabeledButton } from '../../../';
 
+import Library from '../Library';
```

Comment:
I believe the newline above should be removed. Both exports starts with `'../[...]'`, so they should be in the same section.